packages/database/src/migrations/20240319010222-RemoveDisasterTables-modifies-schema.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  return db.runSql(`
+    DROP TABLE IF EXISTS "disasterEvent";
+    DROP TABLE IF EXISTS "disaster";
```

Comment:
Can we drop the user defined types too:

```suggestion
    DROP TABLE IF EXISTS "disasterEvent";
    DROP TABLE IF EXISTS "disaster";
    DROP TYPE IF EXISTS disaster_type;
    DROP TYPE IF EXISTS disaster_event_type;
```